.github/workflows/main.yaml

```diff
@@ -84,4 +84,43 @@ jobs:
           files: ./go_coverage.txt,./lcov_output.info
           name: ubuntu-${{ matrix.ubuntu }}-unittests
           fail_ci_if_error: true
-
+  windows:
+    name: Build with MSYS
+    runs-on: windows-2019
+    timeout-minutes: 90
```

Comment:
@matrixji I am curious about the reason for setting timeout-minutes for `90`, 30 minutes longer the run on ubuntu without running ut. Does compile on windows  much slower than ubuntu?

Reply:
Currently, it needs 60+ mins, this is not suitable for the check pipeline. The main root cause is compiling rocksdb under MinGW is pretty long as ccache is not working, I'll try some patches to accurate the build process before this PR final Merged.

Reply:
>  I'll try some patches to accelerate the build process before this PR Merged.
That would be nice. 👍

Reply:
Now compile milvus with system bundled `arrow` and `rocksdb`,  only take 30+ mins for a fresh build, and only need 10+ mins for cached build.